### PR TITLE
Implement zrevrank

### PR DESCRIPTION
**Pull request by Braxilior**

### CODE REVIEW

File: src/M6Web/Component/RedisMock/RedisMock.php [hunk 1]
```diff
@@ -1048,6 +1048,17 @@ public function zrank($key, $member)
...
     }
 
+    public function zrevrank($key, $member)
+    {
+        $rank = $this->zrank($key, $member);
+        if ($rank === null) {
+            return $this->returnPipedInfo(null);
+        }
+
+        $revRank = count (self::$dataValues[$this->storage][$key]) - $rank - 1;
```
Comment (b-viguier) [inline src/M6Web/Component/RedisMock/RedisMock.php]:
```suggestion
        $revRank = count(self::$dataValues[$this->storage][$key]) - $rank - 1;
```

File: src/M6Web/Component/RedisMock/RedisMock.php [hunk 2]
```diff
@@ -1048,6 +1048,17 @@ public function zrank($key, $member)
...
+        return $this->returnPipedInfo($revRank);
```
Comment (b-viguier) [inline src/M6Web/Component/RedisMock/RedisMock.php]:
```suggestion

        return $this->returnPipedInfo($revRank);
```

**Comment by Braxilior** [comment]
@omansour I have already pushed a commit with the requested changes: https://github.com/M6Web/RedisMock/pull/88/commits/f3239cfc59a9f43cccd4e9eb0993bcc39eff63f8

Is there something wrong ? Sorry, I am not used to making pull requests.

**Comment by omansour** [comment]
v4.7.0

**Pull request merged**